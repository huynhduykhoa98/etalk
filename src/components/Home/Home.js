/* eslint-disable linebreak-style */
/* eslint-disable class-methods-use-this */
/* eslint-disable jsx-a11y/alt-text */
import React, { Component } from 'react';
import { FormattedMessage } from 'react-intl';

class Home extends Component {
  render() {
    return (
      <section id="welcome">
        <div className="container">
          <img src="./images/spider-logo.svg" width={210} />
          <h1>
            <FormattedMessage id="brandMainText" />
          </h1>
          <h2>
            <span>
              <FormattedMessage id="brandChangeText" values={{ order: 1 }} />
            </span>
            <span>
              <FormattedMessage id="brandChangeText" values={{ order: 2 }} />
            </span>
            <span>
              <FormattedMessage id="brandChangeText" values={{ order: 3 }} />
            </span>
          </h2>
        </div>
      </section>
    );
  }
}

export default Home;
