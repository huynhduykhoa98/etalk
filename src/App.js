/* eslint-disable quotes */
/* eslint-disable class-methods-use-this */
import React from 'react';
import { IntlProvider } from 'react-intl';
import Home from './components/Home/Home';
import Navbar from './components/Navbar/Navbar';
import About from './components/About/About';
import Service from './components/Service/Service';
import Technology from './components/Technology/Technology';
import Client from './components/Client/Client';
import Career from './components/Career/Career';
import Contact from './components/Contact/Contact';
import CompanyInfo from './components/CompanyInfo/CompanyInfo';
import Footer from './components/Footer/Footer';
import Parallax from './components/Parallax/Parallax';
import data from './data.json';

class App extends React.Component {
  render() {
    return (
      <IntlProvider locale="en" messages={data}>
        <div>
          <Home />
          <Navbar />
          <Parallax bg="./images/spider-office-1.jpg" />
          <About />
          <Parallax bg="./images/spider-office-2.jpg" />
          <Service />
          <Parallax bg="./images/spider-office-3.jpg" />
          <Technology />
          <Parallax bg="./images/spider-office-4.jpg" />
          <Client />
          <Parallax bg="./images/spider-office-5.jpg" />
          <Career />
          <Parallax bg="./images/spider-office-6.jpg" />
          <Contact />
          <Parallax bg="./images/spider-office-7.jpg" />
          <CompanyInfo />
          <Footer />
        </div>
      </IntlProvider>
    );
  }
}

export default App;
