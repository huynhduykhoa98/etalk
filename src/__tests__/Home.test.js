/* eslint-disable linebreak-style */
import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Home from '../components/Home/Home';
import App from '../App';

Enzyme.configure({ adapter: new Adapter() });
it('test render', () => {
  const wrapped = shallow(<App />);
  expect(wrapped.find(Home).length).toEqual(1);
});
