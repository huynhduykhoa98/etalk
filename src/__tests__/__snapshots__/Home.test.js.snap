// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<section
  id="welcome"
>
  <div
    className="container"
  >
    <img
      src="/images/spider-logo.svg"
      width={210}
    />
    <h1>
      SPIDER
    </h1>
    <h2>
      <span>
        Web Development
      </span>
      <span>
        IT Outsourcing
      </span>
      <span>
        Beyond The Web
      </span>
    </h2>
  </div>
</section>
`;
